Tests/test_GenBank.py

```diff
@@ -136,8 +144,8 @@ def perform_record_parser_test(self, record, length, locus, definition, accessio
     def test_record_parser_01(self):
         path = 'GenBank/noref.gb'
         handle = open(path, "r")
-        gb_iterator = GenBank.Iterator(handle, self.rec_parser)
```

Comment:
The choice of variable name here was deliberately emphasising that ``records`` was not a list of records, but a single-use iterator. Perhaps this concept is now widely enough used in Python that does not need to be stressed?

Reply:
Iterators have been part of Python since version 2.3, released in July 2003. Should be OK to assume that people are familiar with them.

Reply:
I remember the idea being quiet novel to me at the time (coming from languages without anything similar). But OK, this code isn't aimed at Python novices.